rules/bridges.yaml

```diff
@@ -39,13 +40,8 @@ CABHUT:
 
 ^LowBridge:
 	Inherits: ^LowBridgeRamp
```

Comment:
Wrong, it inherits from ^LowBridgeRamp which inherits from ^FeatureTargettypeBuilding. But yes, It doesn't make sense to be able to forcefire something which doesn't have health.